src/api/run_cucumber.ts

```diff
@@ -25,7 +26,7 @@ export async function runCucumber(
   onMessage?: (message: Envelope) => void
 ): Promise<IRunResult> {
   const { cwd, stdout, stderr, env } = mergeEnvironment(environment)
-  const logger = new Console(stdout, stderr)
+  const logger = new Console(stderr)
```

Comment:
This was an oversight when initially added. This `logger` is generally used for `warn` and `error`, but even for `log` (if used) it should still go to `stderr` - only formatters get to write stuff to `stdout`.